<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Object_List</class>
 <widget class="QDockWidget" name="Object_List">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>425</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>425</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>425</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="styleSheet">
   <string notr="true">QDockWidget
{
	color: rgb(255, 255, 255);
}
QDockWidget::title{
			
text-align: center; /* align the text to the left */
			
background: rgb(44, 62, 80);
padding-left: 5px;
}</string>
  </property>
  <property name="windowTitle">
   <string>Object List</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QFrame" name="testFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>42</y>
      <width>279</width>
      <height>360</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>279</width>
      <height>360</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>279</width>
      <height>360</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>1</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>279</width>
      <height>232</height>
     </size>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="selectAllBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>85</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>85</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>85</width>
      <height>25</height>
     </size>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>44</red>
          <green>62</green>
          <blue>80</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	color: rgb(255, 255, 255);
	
}
QToolButton { /* all types of tool button */
    border: 2px solid #8f8f91;
    border-radius: 6px;
   /*background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:1 rgba(, 255));*/
	background-color: rgb(44, 62, 80,255);
}</string>
    </property>
    <property name="text">
     <string>Select All</string>
    </property>
   </widget>
   <widget class="QToolButton" name="deselect_Btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>85</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>85</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>85</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
	color: rgb(255, 255, 255);
	background-color: rgb(44, 62, 80);
}
QToolButton { /* all types of tool button */
    border: 2px solid #8f8f91;
    border-radius: 6px;
    /*background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);*/
	background-color: rgb(44, 62, 80,255);
}</string>
    </property>
    <property name="text">
     <string>Select None</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>35</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>35</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
